### PR TITLE
Adds max-lines rule to configuration

### PR DESCRIPTION
**Pull request by bradbyte**
I'm assuming we shouldn't merge this until I finish the PRs for the repos that use this?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,6 +42,11 @@ module.exports = {
     "no-dupe-keys": 2,
     "no-underscore-dangle": 0,
     "no-undef": 2,
+    "max-lines": ["error", {
+      "max": 300,
+      "skipBlankLines": true,
+      "skipComments": true
```
Comment (brianium) [inline index.js]: Was this what we settled on? Or it just seemed good to skip blank lines and comments? I would include both of these, but I might be the minority 😄

**Review by brianium — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (bradbyte) [reply index.js]: I feel like these were the defaults... going off the list of files that were provided in the Trello story (I think it was more than I commented), it should probably be removed though.

Comment (bradbyte) [reply index.js]: I was wrong, thanks for pointing that out!

**Comment by brianium** [comment]
@bradbumbalough also would you mind incrementing the version in package.json? Maybe 1.1.0? - not sure semver provides a ton of value here - but some increment should be used for npm

**Pull request merged**